lib/PGresponsegroup.pm

```diff
@@ -19,7 +19,7 @@ use strict;
 use Exporter;
 use PGUtil  qw(not_null) ;
 use PGanswergroup;
-use Tie::IxHash;
+# use Tie::IxHash;
```

Comment:
If `Tie::IxHash` is not needed, would it be better to remove this line than comment it out?